tachyon/crypto/hashes/sponge/poseidon/poseidon_grain_lfsr.h

```diff
@@ -101,8 +102,11 @@ PoseidonGrainLFSR<F>::PoseidonGrainLFSR(const PoseidonGrainLFSRConfig& config)
 }
 
 template <typename F>
-std::bitset<F::kModulusBits> PoseidonGrainLFSR<F>::GetBits(size_t num_bits) {
-  std::bitset<F::kModulusBits> ret;
+auto PoseidonGrainLFSR<F>::GetBits(size_t num_bits) {
```

Comment:
Do you have any reason to use `auto` here?

Reply:
Otherwise, `F::kModulusBits` should be changed to `std::conditional_t<math::FiniteFieldTraits<F>::kIsPackedPrimeField, typename math::FiniteFieldTraits<F>::PrimeField, F>::kModulusBits`!